.github/workflows/node-ci.yml

```diff
@@ -1,22 +1,38 @@
 name: Node.js CI
-"on":
+'on':
   push:
     branches:
       - master
-      - dependabot/**
+      - renovate/**
   pull_request:
     types:
       - opened
       - synchronize
 jobs:
+  verify-matrix:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node:
+          - 16.0.0
+          - 18
```

Comment:
defined to ensure that the supported node version range passes verification regardless of what the "preferred" version is defined as for development in the `.nvmrc`